projects/fhi-angular-highcharts/package.json

```diff
@@ -1,16 +1,16 @@
 {
   "name": "@folkehelseinstituttet/angular-highcharts",
-  "version": "1.0.0",
+  "version": "1.1.0",
   "peerDependencies": {
     "@angular/common": "^17.0.0",
     "@angular/core": "^17.0.0",
-    "@folkehelseinstituttet/style": "^5.0.0",
+    "@folkehelseinstituttet/style": "^5.9.0",
```

Comment:
Does 1.1.0 need /style 5.9.0 and BS 5.3.2?

Reply:
Nope, this is not for v1.1.0, but prepping for v2

Reply:
Ah, sorry, of course